app/Services/User/LoginUserService.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace App\Services\User;
+
+use App\Http\Requests\Api\Auth\LoginRequest;
+use App\Http\Resources\ErrorResource;
+use App\Http\Resources\SuccessResource;
+use App\Http\Resources\UserResource;
+use App\Repositories\UserRepository;
+use Exception;
+use Illuminate\Http\Response;
+use Illuminate\Support\Facades\Hash;
+
+class LoginUserService
+{
+    protected $userRepository;
+
+    public function __construct(UserRepository $userRepository)
+    {
+        $this->userRepository = $userRepository;
+    }
+
+    public function execute(LoginRequest $request)
+    {
+        $user = $this->userRepository->findWhere(['email' => $request->input('email')])->first();
+
+        if (!$user || !$this->checkPassword($request->input('password'), $user->password)) {
+            return response([
+                'message' => ['These credentials do not match our records.']
+            ], 404);
+        }
+
+        try {
+            $token = $user->createToken('authToken')->plainTextToken;
+            return new SuccessResource(Response::HTTP_CREATED, "Logged In successfully.", [
```

Comment:
This is 200 status code not 201. 
I mention that because I see you doing this a lot. and it might cause some issue with front end guys since they are expecting 200 rather than 201.

Reply:
Thank you, nice remark, resolved